src/test/java/com/google/cloud/tools/appengine/operations/AppYamlProjectStagingTest.java

```diff
@@ -381,28 +438,48 @@ public void testFindRuntime_malformedAppYaml() throws IOException {
     }
   }
 
+  @Test
+  public void testHasCustomEntrypoint_true() throws IOException, AppEngineException {
+    Path file = appEngineDirectory.resolve("app.yaml");
+    Files.write(
+        file,
+        "entrypoint: custom custom".getBytes(StandardCharsets.UTF_8),
```

Comment:
Oh! Now I think probably `entrypoint:` should be a string list, not string? That is, `getEntrypoint()` should return `List<String>`?

Whatever it should be, I think it's worth testing things like `entrypoint:` (null) or `entrypoint: []` (`getEntryoint()` returning null and/or an empty list).

Reply:
So entrypoint in the app.yaml is a "space separated string", but I think @ludoch can verify this?

Reply:
Can you add another test for `entrypoint:`?

Reply:
To verify that using a list fails?

Reply:
No, I just want to confirm `entrypoint:` returns null instead of an empty string.